### DISCUSSION TITLE
Github copilot

### DISCUSSION THREAD

**Discussion by NatanAlencar17** [community/community#124573] [Copilot Conversations]
Na hora de pegar minha licença de estudante para obter o Github copilot não aparece a opção de linkar um documento da minha instituição só a opção para tirar uma foto do meu celular

**Comment by Akash1134** [comment]
Hi @NatanAlencar17 , thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md)] very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
